frontend/src/features/properties/list/PropertyListView.test.tsx

```diff
@@ -1,261 +1,140 @@
   });
 
-  it('Matches snapshot', async () => {
-    setupTests();
-
-    await act(async () => {
-      const { container } = renderPage();
-      expect(container.firstChild).toMatchSnapshot();
-      await waitFor(async () => {
-        expect(container.querySelector('span[class="spinner-border"]')).not.toBeInTheDocument();
-      });
-    });
+  it('matches snapshot', async () => {
+    setupMockApi();
+    const { component, findSpinner } = setup();
+    await waitFor(async () => expect(findSpinner()).not.toBeInTheDocument());
+    expect(component.asFragment()).toMatchSnapshot();
   });
 
-  it('Displays message for empty list', async () => {
-    mockedService.getPropertyList.mockResolvedValueOnce({
-      quantity: 0,
-      total: 0,
-      page: 1,
-      pageIndex: 0,
-      items: [],
-    });
-
-    const { findByText } = renderPage();
+  it('displays message for empty list', async () => {
+    setupMockApi();
+    const {
+      component: { findByText },
+    } = setup();
 
     // default table message when there is no data to display
     const noResults = await findByText(/No rows to display/i);
     expect(noResults).toBeInTheDocument();
   });
 
-  it('Displays export buttons', async () => {
-    setupTests();
-
-    await act(async () => {
-      const { getByTestId, container } = renderPage();
-      expect(getByTestId('excel-icon')).toBeInTheDocument();
-      expect(getByTestId('csv-icon')).toBeInTheDocument();
-      expect(container.querySelector('span[class="spinner-border"]')).not.toBeInTheDocument();
-    });
-  });
-
-  it('Displays edit button', async () => {
-    setupTests();
-
-    await act(async () => {
-      const { getByTestId } = renderPage();
-      expect(getByTestId('edit-icon')).toBeInTheDocument();
-    });
-  });
-
-  it('Displays save edit button, when edit is enabled', async () => {
-    setupTests();
-
-    await act(async () => {
-      const { getByTestId } = renderPage();
-      expect(getByTestId('edit-icon')).toBeInTheDocument();
-      fireEvent(
-        getByTestId('edit-icon'),
-        new MouseEvent('click', { bubbles: true, cancelable: true }),
-      );
-      await waitFor(() => expect(getByTestId('save-changes')).toBeInTheDocument(), {
-        timeout: 500,
-      });
-    });
-  });
-
-  it('Displays save edit button, when edit is enabled', async () => {
-    setupTests();
+  it('displays list of properties', async () => {
+    setupMockApi([mockParcel]);
+    const {
+      component: { findByText },
+    } = setup();
 
-    await act(async () => {
-      const { getByTestId } = renderPage();
-      expect(getByTestId('edit-icon')).toBeInTheDocument();
-      fireEvent(
-        getByTestId('edit-icon'),
-        new MouseEvent('click', { bubbles: true, cancelable: true }),
-      );
-      await waitFor(() => expect(getByTestId('save-changes')).toBeInTheDocument(), {
-        timeout: 500,
-      });
-    });
+    // default table message when there is no data to display
```

Comment:
does this comment make sense?

Reply:
The comment was there from whoever created the tests but I can remove as it is very clear what the test is asserting

Reply:
nvm this is a copy pasta mistake LOL